Dockerfile

```diff
@@ -32,6 +32,8 @@ RUN sh -c "echo deb http://mirrors.kernel.org/ubuntu focal main universe | tee -
 
 RUN amdgpu-install -y --usecase=rocm --no-dkms
 
+RUN chmod 1777 /tmp
```

Comment:
Why do we need this line?

Reply:
> Why do we need this line?

This is just for the access for my local /tmp directory. I will delete it. Thanks.